src/quo2/components/settings/category_list/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns quo2.components.settings.category-list.view
+  (:require [quo2.components.settings.category-list.style :as style]
+            [quo2.components.settings.settings-list.view :refer [badge-icon
+                                                                 browser-context-icon chevron-icon communities-icons right-button]]
```

Comment:
we prefer to do not use :refer, and use aliases

Reply:
Also you do not need to import all the invididual components of these components, you can just import the whole settings-list component as that is all you need and then forward the props to there.
I will send you and example via discord as it will be easier to communicate there.